starters/basic-starter/components/media--image.tsx

```diff
@@ -17,26 +18,40 @@ export function MediaImage({
   height,
   priority,
   sizes,
+  imageStyle,
   ...props
 }: MediaImageProps) {
   const image = media?.image;
 
   if (!image) {
     return null;
   }
+  let sizeProps;
+  let srcURL;
 
-  const sizeProps =
-    layout === 'fill'
-      ? null
-      : {
-          width: width || image.resourceIdObjMeta.width,
-          height: height || image.resourceIdObjMeta.height,
-        };
+  // Use the image style to render an image if specified.
+  if (imageStyle && image.links[imageStyle]) {
```

Comment:
+1 for checking that the `imageStyle` is available in `image.links`.

I'm wondering if we should expand this with few helpful logger messages. We could first check if `imageStyle` exists, then check if `image.links` exists and log a warning that `consumer_image_styles` needs to be installed and enabled for the consumer the application uses. Then if `image.links[imageStyle]` doesn't exist, log a warning that the image style doesn't exist and advice them to configure the consumer to include that image style in the responses for the consumer used by this application.